include/openssl/core_names.h

```diff
@@ -185,6 +185,7 @@ extern "C" {
 #define OSSL_PKEY_PARAM_MASKGENFUNC         "mgf"
 #define OSSL_PKEY_PARAM_MGF1_DIGEST         "mgf1-digest"
 #define OSSL_PKEY_PARAM_MGF1_PROPERTIES     "mgf1-properties"
+#define OSSL_PKEY_PARAM_TLS_ENCODED_PT      "tls-encoded-pt"
```

Comment:
This should be documented.

Reply:
Is this really specific for TLS, btw?

Reply:
> Is this really specific for TLS, btw?

Yes, and no. I think all of the existing formats are fairly generic formats that might apply to other things. However its possible that there could be a TLS specific format introduced for some future algorithm that doesn't apply to anything else. So I tend to think it should be kept TLS speicfic.

Reply:
This is now documented.